File: PL038.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

double findAvg(int arr[], int n)
{

    double avg = 0, sum = 0;
    for (int i = 0; i < n; i++)
    {
        sum += arr[i];
    }

    avg = sum / n;
    cout << fixed << setprecision(2);
    return avg;
}

int main()
{
    int n = 6;
    int arr[] = {12, 24, 435, 54, 3, 4};
    cout << "Average is : " << findAvg(arr, n) << endl;
    return 0;
}

File: PL038.java
public class PL038 {
    public static void main(String[] args) {
        int n = 6;
        int[] arr = { 2, 34, 2, 32, 4, 12 };

        System.out.println("Average is : " + average(arr, n));
    }

    public static String average(int A[], int N) {
        double avg;
        double sum = 0;
        for (int i : A) {
            sum += i;
        }

        avg = (double) (sum / N);
        String result = String.format("%.2f", avg);
        return result;
    }
}
